### PR TITLE
[PLAY-443]  Body kit style <b>, <strong>, and <a>

### PR DESCRIPTION
**Pull request by Israel-Molestina**
#### Screens

Screen Shot 2023-01-11 at 11 52 07 AM

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-443)

#### How to test this

To test, make sure b strong and a tags within a Body tag have correct styling

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_body/_body.scss [hunk 1]
```diff
@@ -12,13 +12,22 @@
...
   }
-
   @each $dark_color_name, $dark_color_value in $pb_dark_body_colors{
     &[class*=_#{$dark_color_name}][class*=dark]{
       @include pb_body($dark_color_value);
     }
   }
+  b, strong {
+    letter-spacing: -.03em;
+    font-weight: 700;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_body/_body.scss:22]: 2 things:

1. Always use our tokens if they are available. If they are not question why they are not and get them created.
2. Why not just include the styling for title 4? [Import this](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/_titles.scss) and add `@include pb_title_4;`

File: playbook/app/pb_kits/playbook/pb_body/_body.scss [hunk 2]
```diff
@@ -12,13 +12,22 @@
...
+  }
 
+  a {
+    &:hover {
+      color: $text_lt_default;
+    }
+    color: $primary;
+  }
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_body/_body.scss:30]:
```suggestion
  a {
    color: $primary;
    &:hover {
      color: $text_lt_default;
    }
  }
```

Comment (jasoncypret) [reply playbook/app/pb_kits/playbook/pb_body/_body.scss:30]: This is easier to read

File: playbook/app/pb_kits/playbook/pb_body/docs/_body_styled.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Playbook styles the `b`, `strong` and `a` tags within the body kit.
+* `b` and `strong` tag styles: `font-weight: 700` and `letter-spacing: -0.03`
+* `a` tag styles:  `color: $primary` and `color: $text_lt_default` on hover.
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_body/docs/_body_styled.md:3]: Can we just say it's styled like the title size 4? We don't do this breakdown of the styles in any other docs. I don't suggest doing it here. What if we change the letter spacing styles...then we have to remember to update the docs?

**Review by jasoncypret — CHANGES_REQUESTED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**